docs/sc/01_introducing-obol-managers.md

```diff
@@ -31,29 +42,48 @@ contract WithdrawalRecipientOwnable is Ownable {
 
 ```
 
-An Ownable Withdrawal Recipient is the most basic type of withdrawal recipient contract. It implements Open Zeppelin's `Ownable` interface and allows one address to call the `withdraw()` function, which pulls all ether from the address into the owners address (or another address specified). Calling withdraw could also fund a fee split to the Obol Network, and/or the protocol that has deployed and instantiated this DV.
+An Ownable Withdrawal Recipient is the most basic example of a withdrawal recipient contract. It implements Open Zeppelin's `Ownable` interface and allows one address to call the `withdraw()` function, which pulls all ether from the address into the owner's address (or another address specified). This contract does no accounting on the amount of ether that is withdrawn, nor does it differentiate reward from principal.
 
-### Immutable Withdrawal Recipient
+### Optimistic Withdrawal Recipient
 
-An immutable withdrawal recipient is similar to an ownable recipient except the owner is hardcoded during construction and the ability to change ownership is removed. This contract should only be used as part of a larger smart contract system, for example a yearn vault strategy might use an immutable recipient contract as its vault address should never change. 
+This is the primary withdrawal recipient Obol uses, as it allows for the separation of reward from principal, as well as permitting the ongoing withdrawal of accruing rewards.
```

Comment:
should it be "primary" or "consensus layer"

Reply:
I meant it to mean that we don't actually use the Ownable Withdrawal Recipient at all, and that it is only there as a primer to give people an idea how the more complex ones work. We could arguably remove the ownable one at some point, but i thought it was clean so at the time left it in. Is that a bad idea?